src/kite/base.js

```diff
@@ -204,6 +208,7 @@ class BaseKite extends Emitter {
 
   onError(err) {
     this.emit(Event.error, 'Websocket error!')
```

Comment:
`error` event is being used in Kontrol [here](https://github.com/koding/kite.js/blob/master/src/kontrol/base.js#L45)